packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -32,7 +33,22 @@ export interface RadioGroupState extends RadioGroupProps {
   buttonKeys?: string[];
 }
 
-export interface RadioGroupTokens extends IForegroundColorTokens, FontTokens {}
+export interface RadioGroupTokens extends IForegroundColorTokens, FontTokens {
+  /**
+   * Color of required indicator
+   */
+  requiredColor?: ColorValue;
+
+  /**
+   * Amount of padding between the end of the label and the start of the required text
+   */
+  requiredPadding?: ViewStyle['padding'];
+
+  /**
+   * States that can be applied to a RadioGroup
+   */
+  required?: RadioGroupTokens;
```

Comment:
Why does required need to have its own state?

Reply:
Thanks for catching this! I was using this to set the required colors before I created a separate slot for it.